.github/pull_request_template.md

```
@@ -0,0 +1,342 @@
+- [ ] Utilizar hashes para verificar la integridad del código interpretado, bibliotecas, ejecutables y archivos de configuración.
+- [ ] Usar locks o mecanismos de sincronización para evitar condiciones de carrera.
+- [ ] Proteger las variables y recursos compartidos de accesos concurrentes.
+- [ ] Inicializar explícitamente todas las variables y almacenes de datos.
+- [ ] Las aplicaciones que requieran privilegios elevados deben solicitarlos lo más tarde posible y liberarlos lo antes posible.
+- [ ] Evitar errores de cálculo comprendiendo cómo el lenguaje maneja operaciones matemáticas y representaciones numéricas.
+- [ ] No utilizar datos proporcionados por el usuario para ejecutar funciones dinámicamente.
+- [ ] Revisar todas las aplicaciones secundarias, código de terceros y bibliotecas para validar su funcionamiento seguro.
+- [ ] Implementar mecanismos seguros para las actualizaciones automáticas utilizando firmas criptográficas.
+
+
+Certifico que mi solicitud aplica las tecnicas y tacticas de seguridad y de  ingeniería de software para mitigar los ataques comunes: 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)